run.py

```diff
@@ -0,0 +1,4 @@
+from ons_ras_common import ons_env
```

Comment:
Probably worth having a discussion at some point re; how micro-services are launched, doesn't worry me one way or another but it would be good to have a standard approach. I think the current mechanism was borne of Lewis's standards and paths required for the code generator, rather than a specific must-have ...